store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/operation/hdfs/handler/job/factory/AccumuloAddElementsFromHdfsJobFactoryTest.java

```diff
@@ -415,12 +420,8 @@ private JobConf createLocalConf() {
         return conf;
     }
 
-    private java.nio.file.Path Paths(java.nio.file.Path tempDir, String inputDir) {
-        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
-    }
-
     public static final class TextMapperGeneratorImpl extends TextMapperGenerator {
-        public TextMapperGeneratorImpl() {
+        TextMapperGeneratorImpl() {
```

Comment:
Might be worth making this inner class private, it's only used in this class. The visibility of the constructor is less important then.